containers/RoundSection/index.js

```diff
@@ -67,13 +25,80 @@ const RoundSection = ({
   btnStyle,
   sectionSubTitle,
   cardArea,
-  btnTitle,
+  btnTitle
 }) => {
+  const [hasRound, setHasRound] = useState(true);
+  const [endTime, setEndTime] = useState(0);
+  const [isLoading, setLoading] = useState(true);
+
+  async function fetchData() {
+    const res = await fetch("https://api.acquity.io/v1/round/active");
+    if (res!==null){
+    res
+      .json()
+      .then(res => {
+        if (!res["is_concluded"]) {
```

Comment:
Prefer using dot notation since we already know the key: `!res.is_concluded` (Damn snake cases, but nvm don't need to install a library for just this one thing). But for future reference, can try using libraries like [humps](https://www.npmjs.com/package/humps) to convert backend snake_case keys to frontend camelCase keys.